### PR TITLE
Highlight my planets plugin

### PR DESCRIPTION
**Pull request by BenHen75**

### CODE REVIEW

File: content/casual/highlight-my-planets/plugin.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    // paint bigger ones first
+    planets.sort((a, b) => b.range - a.range);
+
+    for (const p of planets) {
+
+      // draw range circle
+      const { x, y } = viewport.worldToCanvasCoords(p.location.coords);
+      const hsl = this.hexToHsl(this.ownColor);
+
+      const fac = Math.max(0, Math.log2(this.rangePercent / 5));
```
Comment (modukon) [inline content/casual/highlight-my-planets/plugin.js:133]: What do you think about replacing this line with:
`const fac = this.rangePercent*0.01;`
and setting the this.rangePercent default to 20 (in the constructor)

**Comment by modukon** [comment]
Ideas for the future (i would implement them if people are ok with it and nobody else wants to do it)

- Instead of the color picking option planets have different colors depending on their type.
I think this is how PlanetTypes can be imported:
https://github.com/darkforest-eth/plugins/blob/68f6b35f9a606c5fd9524d451d46372bb92f4aab/content/productivity/quick-upgrade/plugin.js#L28

- Clicking a planet from a different player switches the plugin to highlight that persons planets

File: content/casual/highlight-my-planets/plugin.js (hunk 1 above)
Comment (BenHen75) [reply content/casual/highlight-my-planets/plugin.js]: Thanks for the review, you right, it's done

**Review by modukon — APPROVED** [review]

**Review by modukon — APPROVED** [review]

**Comment by BenHen75** [comment]
- I like ​​having a color Picker, but the idea of ​​being able to differentiate the planets is also good, I can add an option to let the user choose
- [heatmap](https://github.com/darkforest-eth/plugins/blob/master/content/casual/heatmap/plugin.js) plugin already allows you to compare your empire with other players

**Comment by modukon** [comment]
> * I like ​​having a color Picker, but the idea of ​​being able to differentiate the planets is also good, I can add an option to let the user choose
> * [heatmap](https://github.com/darkforest-eth/plugins/blob/master/content/casual/heatmap/plugin.js) plugin already allows you to compare your empire with other players

My idea was if you click on another planet from another player, it will stop highlighting your planets and highlight theirs until you unselect that planet or click on a planet from a different person.
Its not about comparing your empire with others, but seeing only the planets of a specific player so you can also quickly see where are all their small planets.

**Comment by jacobrosenthal** [comment]
Maybe we can share some of the heatmap code eventually. 
Also modukon's usecase sounds interseting, but can fork or pr that

**Pull request merged**